packages/dialtone-vue2/.storybook/preview.jsx

```diff
@@ -1,20 +1,25 @@
 import '../css/dialtone-globals.less';
 import '@dialpad/dialtone-css/lib/dist/dialtone.css';
+import { addons } from '@storybook/preview-api';
 import { setTheme } from '@/../../common/themes/config';
 import DpLight from '@/../../common/themes/dp-light.js';
+import DpDark from '@/../../common/themes/dp-dark.js';
 import { MINIMAL_VIEWPORTS } from '@storybook/addon-viewport';
 import { DocsContainer } from '@storybook/addon-docs';
-import { useDarkMode } from "storybook-dark-mode";
+import { useDarkMode, DARK_MODE_EVENT_NAME } from "storybook-dark-mode";
 import Vue from 'vue';
-import React from 'react';
```

Comment:
Don't remove this react import, seems like it is not being used here, but it breaks the storybook preview window if removed.

Reply:
Ah yeah, thought that might be there for a reason